packages/fxa-content-server/tests/functional/pages.js

```diff
@@ -13,29 +13,14 @@ var pages = [
   '',
   'authorization',
   'boom',
-  // 'cannot_create_account',
```

Comment:
Sorry if my comment [here](https://github.com/mozilla/fxa/pull/15340/#discussion_r1201321288) was unclear. We should remove the commented out tests, I just think we should leave this comment above for when we comment out tests in the future:

> // Commented out pages here likely mean the React version is rolled out to 100%
> // and we don't want to check these for `#stage header` selectors.